tetris.js

```diff
@@ -344,12 +344,61 @@ module.exports = {
 		}
 	},
 	determineShift: function (typeOfTetromino, rotationPhaseOfTetromino) {
-		return { x: 0, y: 0 };
+		if(rotationPhaseOfTetromino === "stable"){
```

Comment:
Please make sure all ifs have the following structure:
Outer if checks for tetromino type
Inner if checks for phase type